### PR TITLE
feat: move icons to icons.md from readme.md

### LINKED ISSUE #1195: [DOCS] Move the supported Icons to a seprate file.
### Description

- Move the [Available Icons](https://github.com/EddieHubCommunity/LinkFree#available-icons) section from the README to a file name `icons.md`

- Also, add a message in the README for the contributors to check the available icon file.
It can something like this, and hyperlink the icon's file.
_"Check [`Icons.md`]() for the all the supported icons"_

### Screenshots

Section to be moved
Screenshot from 2022-03-23 09-59-56

### PR DESCRIPTION
**Pull request by kishoreandra**
Move icons from readme.md to a separate file (icons.md) and add link to navigate to the same from readme.

Fixes #1195 

## Changes proposed

- [x]  Move icons to seperate file; marked as done

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1200"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,25 +101,13 @@ Your URL will be `http://linkfree.eddiehub.org/<yourusername>`
...
-| Telegram   | <img src="https://user-images.githubusercontent.com/65664185/138503468-8f27e3a9-d9ad-4348-85a4-d2c1761cd81f.png" width=100% height=30%>  | TikTok         | <img src="https://user-images.githubusercontent.com/76241195/147094674-a0b67f90-c62e-4162-93b5-dd313611ff28.png" width=100% height=30%>  | Twitch    | <img src="https://user-images.githubusercontent.com/76241195/147094354-12d3b38d-bb87-4c52-a3b7-4d934498da31.png" width=100% height=30%>  |
-| Twitter    | <img src="https://user-images.githubusercontent.com/65664185/138503209-1ce0ebbc-5590-4940-8cd0-2dadacbf09ed.png" width=100% height=30%>  | Vimeo          | <img src="https://user-images.githubusercontent.com/65664185/138503257-6af44a9c-c81a-4657-b182-6a991157810f.png" width=100% height=30%>  | YouTube   | <img src="https://user-images.githubusercontent.com/65664185/138503305-ff60cf54-6b0b-4e18-9446-b4f6869b9511.png" width=100% height=30%>  |
+Check [Icons.md](/icons.md) for the all the supported icons
```
Comment (schmelto) [inline README.md:110]:
```suggestion
Check [Icons.md](/icons.md) for all the supported icons.
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]
Lgtm

**Review by vikasganiga05 — APPROVED** [review]
LGTM :+1: Thanks! for your contribution :tada:

**Review by ssensalo — APPROVED** [review]

**Comment by kishoreandra** [comment]
@vikasganiga05 @schmelto 
Great 😃 Thank you

**Comment by schmelto** [comment]
> @vikasganiga05 @schmelto 
> 
> Great 😃 Thank you, what would be the next steps 🤔 

Waiting for another maintainer to approve and than the pr will get merged. No need from you side to do something.

**Comment by kishoreandra** [comment]
> > @vikasganiga05 @schmelto
> > Great 😃 Thank you, what would be the next steps 🤔
> 
> Waiting for another maintainer to approve and than the pr will get merged. No need from you side to do something.

Awesome 😎

**Review by emmalearnscode — APPROVED** [review]
Looks good, thanks for contributing!

**Comment by kishoreandra** [comment]
> Looks good, thanks for contributing!

Thank you 😊, my pleasure

**Pull request merged**